app/build.gradle

```diff
@@ -216,6 +213,7 @@ dependencies {
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0-rc03'
     implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0-rc03'
     implementation "androidx.work:work-runtime-ktx:2.2.0"
+    implementation "com.google.maps.android:android-maps-utils:2.0.3"
```

Comment:
What's this for?

Reply:
I was getting class not found error on a map related class (the name I couldn't recall now) when I was trying to run the app. Now that I tried without this, it works. Not sure exactly what happened. Removed this line now.